packages/block-editor/src/components/global-styles/shadow-panel-components.js

```diff
@@ -32,6 +32,11 @@ import { unlock } from '../../lock-unlock';
  * @type {Array}
  */
 const EMPTY_ARRAY = [];
+const {
+	CompositeV2: Composite,
+	useCompositeStoreV2: useCompositeStore,
+	CompositeItemV2: CompositeItem,
```

Comment:
Nit: Feel free to reorder to keep alphabetical order. Some nitpicky eyes might find this more readable 😉 

```suggestion
	CompositeItemV2: CompositeItem,
	CompositeV2: Composite,
	useCompositeStoreV2: useCompositeStore,
```